### PR TITLE
Org.getzola.zola

### PR DESCRIPTION
**Pull request by Ayush1325**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
https://github.com/getzola/zola/issues/1572#issuecomment-1156426533

- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

I have created a PR to move Appdata to upstream as well. (https://github.com/getzola/zola/pull/1900)

Additional @Newbytee will co-maintain this.

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.getzola.zola

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
bot, build org.getzola.zola

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "skip-icons-check": true
```
Comment (hfiguiere) [inline flathub.json:2]:
```suggestion
  "skip-icons-check": true,
```

**Comment by Newbytee** [comment]
bot, build org.getzola.zola

**Comment by Newbytee** [comment]
@Ayush1325 Do you think this is ready to merge now?

**Comment by Ayush1325** [comment]
@Newbytee Once the build completes with the updated metadata, it should be fine to merge.

**Comment by barthalion** [comment]
bot, build org.getzola.zola

**Comment by barthalion** [comment]
```
Processing application org.getzola.zola
Error loading AppData file: failed to parse /srv/buildbot/worker/build-x86_64-7/build/.flatpak-builder/rofiles/rofiles-vAuJLv/files/share/appdata/org.getzola.zola.appdata.xml: Error on line 58 char 14: Element “releases” was closed, but the currently open element is “release”
Error: ERROR: appstream-compose failed: Child process exited with code 1
```

**Comment by Newbytee** [comment]
> @Newbytee, I think this is fine to merge now. Unless you want to wait for [getzola/zola#1900](https://github.com/getzola/zola/pull/1900) to get merged

I think it's fine as-is. Even when that gets merged, we'll have to wait for a new release to make use of it here.

**Pull request closed without merging**